ChangeLog.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.6] - 2022-08-16
+### Added
+- Basic auth support for OAuth2. We need to add Base64 encoded Basic Auth header while making OAuth2 request in OAuth2 retrieve token scheme.
+- Removed ConnectorContext from validate creds request
+- Made api version nullable
+- Removed api version from connector runtime settings and added IsSandboxAccount
```

Comment:
Can we reframe this, this has been added in example

Reply:
I have removed sandbox account phrase, its not a feature imo